Examples/Algorithms/Digitization/include/ActsExamples/Digitization/DigitizationConfig.hpp

```diff
@@ -121,17 +116,15 @@ struct DigiComponentsConfig {
 
 class DigitizationConfig {
  public:
-  DigitizationConfig(bool merge, double sigma, bool commonCorner)
-      : DigitizationConfig(merge, sigma, commonCorner,
-                           Acts::GeometryHierarchyMap<DigiComponentsConfig>()) {
-  }
+  DigitizationConfig();
```

Comment:
Not sure if this in the scope of this refactor, but in general, I don't see see the need of all those constructors, since they as far as I see don't contain any code that ensures certain postconditions or so... I think, we could remove them all?